nativescript-core/ui/core/view-base/view-base.d.ts

```diff
@@ -310,7 +310,7 @@ export abstract class ViewBase extends Observable {
     /**
      * Invalidates the layout of the view and triggers a new layout pass.
      */
-    public requestLayout(): void;
+    public requestLayout(calledFromParent?:boolean): void;
```

Comment:
The same here, maybe the parameter name should be **calledFromChild**
```suggestion
    public requestLayout(calledFromChild?:boolean): void;
```